app/views/ai-league/AILeagueView.vue

```diff
@@ -40,6 +40,7 @@ export default {
   },
 
   mounted () {
+    this.fetchData({ componentName: this.$options.name, options: { loadedEventName: 'AI League: Loaded' } })
```

Comment:
i dont see AI League here: https://github.com/codecombat/codecombat/blob/master/app/core/store/modules/teacherDashboard.js#L194 - should we add an option and move .fetchRequiredInitialData part there?